Library/ViewModels/ThanksViewModel.swift

```diff
@@ -77,21 +78,25 @@ public final class ThanksViewModel: ThanksViewModelType, ThanksViewModelInputs,
   public init() {
     let project = self.configureWithDataProperty.signal
       .skipNil()
-      .map(first)
+      .map { $0.project }
 
     let rewardAndCheckoutData = self.configureWithDataProperty.signal
       .skipNil()
       .map { ($0.reward, $0.checkoutData) }
 
-    self.backedProjectText = project.map {
-      let string = Strings.You_have_successfully_backed_project_html(
-        project_name: $0.name
-      )
+    self.backedProjectText = self.configureWithDataProperty.signal
+      .skipNil()
+      .map { project, _, _, _ in
```

Comment:
Fourth arg here will be the pledge total.

Reply:
Is the plan to then gate the string from within this output?

Reply:
@scottkicks Correct; this is just plumbing to get all the data where it needs to go.